.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -7,29 +7,34 @@ assignees: ''
 
 ---
 
-**Describe the bug**
-A clear and concise description of what the bug is.
+## 🐞 Bug description
+<!-- A clear and concise description of what the bug is. -->
 
-**To Reproduce**
+## 🔢 Steps to Reproduce
+<!-- 
 Steps to reproduce the behavior:
 1. Go to '...'
 2. Click on '....'
 3. Scroll down to '....'
-4. See error
+4. See error -->
 
-**Expected behavior**
-A clear and concise description of what you expected to happen.
+## ⚙️ Expected behavior
+<!-- A clear and concise description of what you expected to happen. -->
 
-**Screenshots**
-If applicable, add screenshots to help explain your problem.
+## 🖥️ Screenshots
+<!-- If applicable, add screenshots to help explain your problem a good animated screenshot can replace 'steps to reproduce'. -->
 
-**Desktop (please complete the following information):**
- - OS: [e.g. iOS]
+## 💻 System information
+<!-- Please complete the following information (you can get most of this information from https://whatismybrowser.com) -->
```

Comment:
I'm impressed by `https://whatismybrowser.com)` - it's very good.

Reply:
It's very good, isn't it! :) There are a few others out there too but I think that's the most user-friendly one.